base/multimedia.jl

```diff
@@ -104,6 +104,18 @@ for that case. If a type benefits from custom human-readable output though,
 `show(::IO, ::MIME"text/plain", ::T)` should be defined. For example, the `Day` type uses
 `1 day` as the output for the `text/plain` MIME type, and `Day(1)` as the output of 2-argument `show`.
 
+# Examples
+```jldoctest
+julia> struct Day
+       n::Int
+       end
+
+julia> Base.show(io::IO, ::MIME"text/plain", d::Day) = print(io, "$(d.n) day")
```

Comment:
```suggestion
julia> Base.show(io::IO, ::MIME"text/plain", d::Day) = print(io, "\$(d.n) day")
```
I expect this is the fix you're looking for.